### DISCUSSION TITLE
Reset Auto numbering on demand

### DISCUSSION THREAD

**Discussion by kookma** [KaTeX/KaTeX#3265] [Q&A] [answered]
I use KaTeX with TidllyWIki (https://kookma-katex.tiddlyhost.com/)

One issue I encountered is when I use environment with autonumbering like `\begin{equation}... \end{equation}`, KaTeX nicely autonumber each equation!

Then I start a new section and I expect to be able to reset the autonumbering to start from 1 again! 

Ho can I instruct KaTeX to do that?

![image](https://user-images.githubusercontent.com/830394/133721332-2b68b9d2-56de-4139-84b6-8138126587e2.png)

**Comment by edemaine** [comment, marked as answer]
KaTeX uses CSS counters for those numbers, so you'll want to use [CSS `counter-reset`](https://developer.mozilla.org/en-US/docs/Web/CSS/counter-reset) to reset the counter.  Depending on how your plugin works, a rule like the following might work:
```css
h2 { counter-reset: katexEqnNo; }
```
